ui/src/components/UploadStory.tsx

```diff
@@ -0,0 +1,855 @@
+                  return (
+                    <TextField
+                      {...params}
+                      required
+                      variant="outlined"
+                      label=""
+                      placeholder="Enter where story was written *"
+                    />
+                  );
+                }}
+              />
+              <FormControl>
+                <UploadLabelsText>Year Published</UploadLabelsText>
+                <StyledSelect
+                  variant="outlined"
+                  value={formInput.year ? formInput.year : ""}
+                  onChange={handleChange}
+                  inputProps={{
+                    name: "year",
+                    id: "select-label-year",
+                  }}
+                >
+                  {yearArray.map((year) => (
+                    <StyledMenuItem key={year} value={year}>
+                      {year}
+                    </StyledMenuItem>
+                  ))}
+                </StyledSelect>
+              </FormControl>
+              <FormControl>
+                <UploadLabelsText>Tags</UploadLabelsText>
+                <StyledAutocomplete
+                  autoHighlight
+                  multiple
+                  id="tags-outlined"
+                  name="tags"
+                  freeSolo
+                  options={tagOptions ? tagOptions : [""]}
+                  filterSelectedOptions
+                  onChange={(_, newValue) => setTagArrayValues(newValue)}
+                  value={tagArray ? tagArray : [""]}
+                  renderTags={(value, getTagProps) =>
+                    value.map((option, index) => (
+                      <Chip
+                        key={option}
+                        variant="outlined"
+                        label={option}
+                        {...getTagProps({ index })}
+                      />
+                    ))
+                  }
+                  renderInput={(params: TagParameters) => {
+                    params.inputProps.onKeyDown = handleKeyDown;
+                    return (
+                      <TextField
+                        {...params}
+                        variant="outlined"
+                        label="Shoe Tags"
+                        placeholder="Select Tags"
+                      ></TextField>
+                    );
+                  }}
+                />
+              </FormControl>
+            </StyledBackgroundColor>
+            <StyledBackgroundColor>
+              <UploadStoriesHeading>Multimedia</UploadStoriesHeading>
+              <ImageContainer>
+                {newImage == "" ? (
+                  <div>
+                    <UploadLabelsText>Add Image</UploadLabelsText>
+                    <StyledDropzoneArea
+                      showFileNames
+                      acceptedFiles={["image/*"]}
+                      filesLimit={1}
+                      dropzoneText={"Drag image here or select from device"}
```

Comment:
Make sure to merge in updates from https://github.com/uwblueprint/shoe-project/pull/281 for image size

Reply:
nice catch, ty